mapusaurus/respondents/views.py

```diff
@@ -51,8 +51,10 @@ def select_metro(request, agency_id, respondent):
     """Once an institution is selected, search for a metro"""
     institution = get_object_or_404(Institution, respondent_id=respondent,
                                     agency_id=int(agency_id))
+    year = request.session.get('year', 2014)
```

Comment:
`2014` is scattered throughout the code as the default year if one isn't selected. Because we'll be loading 2015 data next year, we should probably do something like create a `CURRENT_YEAR` variable in `local_settings.py` instead of hardcording it. Something to think about.